Tella/Components/Connections/Card/ConnectionCardDetails.swift

```diff
@@ -8,26 +8,27 @@
 
 import SwiftUI
 
-struct ConnectionCardDetail: View {
+struct ConnectionCardDetailsView: View {
```

Comment:
I think it's better to move `Components/Connections` folder under `CommonConnectionReport`